recipe/meta.yaml

```diff
@@ -75,22 +81,24 @@ requirements:
     - xbyak >=6.69.1  # [x86_64]
-    - libprotobuf-static =3.20.3  # [not win64]
-    - libprotobuf-static  # [win64]
+    - libprotobuf-static =3.20.3
+    - cudnn =8.8.0  # [cuda_compiler_version != "None"]
+    - cutensor =1.6.1.5  # [cuda_compiler_version != "None"]
   run:
     - {{ pin_subpackage('libopenvino', max_pin='x.x.x') }}
-    - {{ pin_subpackage('libopenvino-intel-cpu-plugin', max_pin='x.x.x') }}  # [x86_64]
-    - {{ pin_subpackage('libopenvino-arm-cpu-plugin', max_pin='x.x.x') }}  # [aarch64 or arm64]
-    - {{ pin_subpackage('libopenvino-intel-gpu-plugin', max_pin='x.x.x') }}  # [win64 or linux64]
-    - {{ pin_subpackage('libopenvino-auto-plugin', max_pin='x.x.x') }}
-    - {{ pin_subpackage('libopenvino-hetero-plugin', max_pin='x.x.x') }}
-    - {{ pin_subpackage('libopenvino-auto-batch-plugin', max_pin='x.x.x') }}
-    - {{ pin_subpackage('libopenvino-ir-frontend', max_pin='x.x.x') }}
-    - {{ pin_subpackage('libopenvino-onnx-frontend', max_pin='x.x.x') }}
-    - {{ pin_subpackage('libopenvino-paddle-frontend', max_pin='x.x.x') }}
-    - {{ pin_subpackage('libopenvino-pytorch-frontend', max_pin='x.x.x') }}
-    - {{ pin_subpackage('libopenvino-tensorflow-frontend', max_pin='x.x.x') }}
-    - {{ pin_subpackage('libopenvino-tensorflow-lite-frontend', max_pin='x.x.x') }}
+    - {{ pin_subpackage('libopenvino-intel-cpu-plugin', max_pin='x.x.x') }}             # [cuda_compiler_version == "None" and x86_64]
+    - {{ pin_subpackage('libopenvino-arm-cpu-plugin', max_pin='x.x.x') }}               # [aarch64 or arm64]
+    - {{ pin_subpackage('libopenvino-intel-gpu-plugin', max_pin='x.x.x') }}             # [win64 or linux64]
+    - {{ pin_subpackage('libopenvino-intel-nvidia-plugin', max_pin='x.x.x') }}          # [cuda_compiler_version != "None"]
+    - {{ pin_subpackage('libopenvino-auto-plugin', max_pin='x.x.x') }}                  # [cuda_compiler_version == "None"]
+    - {{ pin_subpackage('libopenvino-hetero-plugin', max_pin='x.x.x') }}                # [cuda_compiler_version == "None"]
+    - {{ pin_subpackage('libopenvino-auto-batch-plugin', max_pin='x.x.x') }}            # [cuda_compiler_version == "None"]
+    - {{ pin_subpackage('libopenvino-ir-frontend', max_pin='x.x.x') }}                  # [cuda_compiler_version == "None"]
+    - {{ pin_subpackage('libopenvino-onnx-frontend', max_pin='x.x.x') }}                # [cuda_compiler_version == "None"]
+    - {{ pin_subpackage('libopenvino-paddle-frontend', max_pin='x.x.x') }}              # [cuda_compiler_version == "None"]
+    - {{ pin_subpackage('libopenvino-pytorch-frontend', max_pin='x.x.x') }}             # [cuda_compiler_version == "None"]
+    - {{ pin_subpackage('libopenvino-tensorflow-frontend', max_pin='x.x.x') }}          # [cuda_compiler_version == "None"]    
+    - {{ pin_subpackage('libopenvino-tensorflow-lite-frontend', max_pin='x.x.x') }}     # [cuda_compiler_version == "None"]
```

Comment:
why do we need to skip all this stuff when build with CUDA?

Reply:
We have two reasons in my opinion:
1st - Not to duplicate the whole content for user who want to proceed with inference on NVIDIA GPU(discussable)
2nd - Because of 'disk space' issue, but even without all this stuff we faced it on both platforms. CUDA dependencies requires a lot of space.

Reply:
how users who wants both NVIDIA and CPU will live?